src/io/config.cpp

```diff
@@ -42,6 +42,19 @@ std::unordered_map<std::string, std::string> Config::Str2Map(const char* paramet
   for (auto arg : args) {
     KV2Map(&params, Common::Trim(arg).c_str());
   }
+  // define verbosity and set logging level
+  int verbosity = Config().verbosity;
+  GetInt(params, "verbose", &verbosity);
+  GetInt(params, "verbosity", &verbosity);
```

Comment:
I believe we should pop `verbose` and `verbosity` here to not process them twice later
https://github.com/microsoft/LightGBM/blob/1cc9f9dcee15586aefdd8271a1c04ad010d3c53a/src/io/config.cpp#L45

Reply:
I wanted to leave them because I'm not sure if they're used some place after but I can drop them here if this is the only place they're used.

Reply:
I left them so that we can warn about duplicates later.